dockerfiles/lint.Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apk add --no-cache gcc musl-dev
 WORKDIR /src
 
 FROM base
+ENV GOFLAGS="-buildvcs=false"
```

Comment:
That should probably do the trick as well; didn't want to add `git` as a dependency for linting if it's not needed.